.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
+	// Use IntelliSense to learn about possible attributes.
+	// Hover to view descriptions of existing attributes.
+	// For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
```suggestion
```
Probably should remove this so GitHub doesn't complain about the comments, which is an addition to JSON that only some JSON parsers implement.
Could mark this as a JSONC file with the `.gitattributes`, but I'm on my phone..